spec/models/section_spec.rb

```diff
@@ -41,4 +41,100 @@
       end
     end
   end
+
+  context 'when sections are created and destroyed' do

```

Comment:
All this really tests is the callback, not that the event was sent. We should either:
a) Unit test EventSender in addition to these tests (the EventSender tests would ensure the correct data is published to redis when EventSender methods are called
b) Change this test to check that the correct data is published to redis when sections / courses are altered.

a) would be an additional unit test while b) would be more of an integration test. My recommendation would be b) in this case as testing that the callbacks are working doesn't test much of our own code because callbacks are built into rails and whats really important here is the end result.